scripts/xmpro-website-scripts/scrape-xmpro-website-latestnews.py

```diff
@@ -0,0 +1,165 @@
+
+        with open(filename, 'w', encoding='utf-8') as file:
+            file.write(f"# {title}\n\n")
+            file.write(f"URL: {url}\n\n")
+            file.write(content)
+        print(f"Content saved to {filename}")
+        return filename
+    except Exception as e:
+        print(f"Error occurred while saving to file: {e}")
+        return None
+
+def get_all_blog_urls(base_url, num_pages):
+    all_blog_urls = set()  # Use a set to avoid duplicate URLs
+    for page_num in range(1, num_pages+1):
+        page_url = f"{base_url}page/{page_num}/"
+        try:
+            headers = {
+                'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/58.0.3029.110 Safari/537.3'}
+            with requests.Session() as session:
+                response = session.get(page_url, headers=headers)
+                response.raise_for_status()  # Raise an exception for bad status codes
+                soup = BeautifulSoup(response.text, 'html.parser')
+                # Find the div with class 'large-9 col'
+                target_div = soup.find('div', class_='large-9 col')
+                if target_div:
+                    # Find all anchor tags within the target div
+                    for link in target_div.find_all('a', href=True):
+                        # Resolve relative URLs to absolute URLs
+                        absolute_url = urljoin(base_url, link['href'])
+                        all_blog_urls.add(absolute_url)
+                else:
+                    print(f"Could not find the target div on page {page_num}")
+        except requests.RequestException as e:
+            print(f"Failed to retrieve content for page {page_num}: {e}")
+        except Exception as e:
+            print(f"Error occurred: {e}")
+        # Introduce a delay of 1 seconds between navigating to a new page
+        sleep(1)
+    return all_blog_urls
+
+def get_max_page_numbers(html, base_url):
+    soup = BeautifulSoup(html, 'html.parser')
+    page_numbers = soup.find_all(class_='page-number')
+    max_page = 1  # Default value if no page numbers are found
+    for page_number in page_numbers:
+        try:
+            page = int(page_number.text)
+            page_url = urljoin(base_url, page_number.get('href'))
+            if page > max_page:
+                max_page = page
+        except ValueError:
+            pass  # Ignore non-integer page numbers
+    return max_page
+
+def generate_readme(files, folder_path):
+    try:
+        readme_content = []
+        for file_info in files:
+            # Remove "docs/" from file path and replace "\" with "/"
+            file_path = file_info['path'].replace("docs/", "").replace("\\", "/")
+            readme_content.append(f"* [{file_info['title']}]({file_path})\n")
+        
+        # Create the README.md file in the same folder as the exported files
+        readme_file_path = os.path.join(folder_path, 'copy-me-latest-news.md')
```

Comment:
@jacksonmcguirkxmpro 

lets call all of these `copy-me.md` so that they are easy to search for all of them at once